appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -943,6 +943,41 @@
   ;;      )
   ;;  '(100 200 17 300))
 
+(define-syntax map_nondest
+  (syntax-rules ()
+    ((_ lambda-arg-name body-form list)
+     (yail-list-map (lambda (lambda-arg-name) body-form) list))))
+
+;;(define-syntax map_nondest
```

Comment:
It looks like this is identical to the above code and it is commented out.  I'm guessing that it is intended to be the commented out definition of `map_dest`.

Reply:
Fixed